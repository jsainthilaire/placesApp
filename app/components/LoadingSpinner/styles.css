.loadingContainer {
    height: 100px;
    width: 100px;
    z-index: 200;
}

.loading {
    animation: load 2s linear infinite;
    border: 10px solid #EFEBE9;
    border-top-color: #8D6E63;
    border-radius: 100%;
    box-sizing: border-box;
    height: 100%;
    width: 100%;
}

@keyframes load {
    from {
        transform: rotate(0deg);
    }
    to {
        transform: rotate(360deg);
    }
}
