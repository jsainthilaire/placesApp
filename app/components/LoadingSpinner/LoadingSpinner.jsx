import React from 'react'
import { loadingContainer, loading } from './styles.css'

const LoadingSpinner = () => (
  <div className={loadingContainer}>
    <div className={loading} />
  </div>
)


export default LoadingSpinner
