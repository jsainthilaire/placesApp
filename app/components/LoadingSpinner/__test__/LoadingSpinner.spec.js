import React from 'react'
import { shallow } from 'enzyme'
import toJson from 'enzyme-to-json'
import LoadingSpinner from '../LoadingSpinner'

test('LoadingSpinner should render correctly', () => {
  const wrapper = shallow(<LoadingSpinner />)
  expect(toJson(wrapper)).toMatchSnapshot()
})
