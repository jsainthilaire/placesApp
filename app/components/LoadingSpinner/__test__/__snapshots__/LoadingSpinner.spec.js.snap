// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`LoadingSpinner should render correctly 1`] = `
<div
  className="loadingContainer"
>
  <div
    className="loading"
  />
</div>
`;
